www/style.css

```diff
@@ -2117,3 +2025,391 @@ dl.mempool-histogram .bar:before {
 .fee-estimates th, .fee-estimates td {
   padding: 0.6rem 0.2rem;
 }
+
+
+
+/*New Design CSS Flexbox */
+/* Main Navigation */
+
+
+.container{
+  width: 100%;
+  max-width: 1200px !important;
+  padding-left: 20px !important;
+  padding-right: 20px !important;
+}
+
+.nav-container{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 0;
+}
+
+.main-nav-container{
+  position: relative;
+  display: flex;
+  align-items: center;
+}
+
+.main-nav{
+  margin: 0;
+  padding: 0;
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: flex-end;
+  position: relative;
+}
+
+.main-nav li{
+  text-decoration: none;
+  list-style: none;
+  margin-right: 30px;
+}
+
+.main-nav li a{
+  display: flex;
+  color: #ffffff;
+  align-items: center;
+  justify-content: center;
+}
+
+.main-nav li.active a{
+  border: 1px solid #ffbb00;
+}
+
+
+.main-nav .menu-logo{
+  height: 22px;
+  margin-right: 8px;
+  margin-bottom: 2px;
+}
+
+
+/* Menu Toggle Icon -- Toggle Style and Animation*/
+
+.toggle-container{
+  position: relative;
+  display: flex;
+  padding: 30px 0 30px 30px;
+}
+
+.toggle-container .burger-icon {
+  position: relative;
+  height: 20px;
+  width: 30px;
+  margin: 0;
+  z-index: 5;
+}
+
+.toggle-container .burger-icon span {
+  position: absolute;
+  width: 100%;
+  height: 2px;
+  top: 50%;
+  left: 0;
+  display: block;
+  background: #78838e;
+  transition: .5s;
+}
+
+.toggle-container .burger-icon span:first-child{
+  top: 3px;
+}
+
+.toggle-container .burger-icon span:last-child{
+  top: 17px;
+}
+
+.toggle-container .burger-icon:hover{
+  cursor: pointer;
+}
+
+.toggle-container .toggle-menu{
+  font-size: 14px;
+  width: 700px;
+  height: 0;
+  visibility: hidden;
+  position: absolute;
+  top: 70px;
+  right: 0;
+  z-index: 10;
+  transition: .1s;
+  border-radius: 8px;
+  box-shadow: 0 20px 40px black;
+  background-image: radial-gradient(circle at 7% 50%, #1c2127, #15181c 80%);
+}
+
+/**** 
+  * Burger Menu Hover works on Bigscreen only 
+  * Hover is used for displaying the dropdown menu to support NO-JS
```

Comment:
This is still technically an issue, but now only for screens <500px rather than <900px, which may be acceptable?

NoScript users are typically on desktop, which should rarely ever be so small, even with a split screen.